app/presenters/flood_risk_engine/tabular_enrollment_detail/row_builder.rb

```diff
@@ -65,6 +65,13 @@ def correspondence_contact_telephone_row
                   value: enrollment_presenter.correspondence_contact_telephone_number
       end
 
+      def registration_number_row
+        return unless enrollment_presenter.organisation_registration_number
+        build_row name: :company_number,
+                  value:  enrollment_presenter.organisation_registration_number,
+                  step: "#{organisation_type}_number"
+      end
+
       private
```

Comment:
I wonder if organisation_registration_number is more consistent? Otherwise its a bit ambiguous as to what a registration is.